tools/building.py

```diff
@@ -319,6 +319,10 @@ def configure(args, stdout=None, stderr=None, env=None, cflags=[], **kwargs):
     # compilation with emcc, but instead do builds natively with Clang. This
     # is a heuristic emulation that may or may not work.
     env['EMMAKEN_JUST_CONFIGURE'] = '1'
+    if not any(a.startswith('--prefix') for a in args):
+      sysroot = os.path.join(shared.Cache.get_sysroot_dir())
+      args.append('--prefix=' + os.path.join(shared.Cache.get_sysroot_dir(), 'local'))
```

Comment:
this code seems unrelated to the comment above it, and i'm not sure what it does?

Reply:
You are right, this is kind of a separate change.. and it requires some more testing.